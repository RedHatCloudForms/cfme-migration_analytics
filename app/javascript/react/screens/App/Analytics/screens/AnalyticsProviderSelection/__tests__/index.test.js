// TODO add unit tests for index.js

it('works', () => expect(true).toBe(true));
