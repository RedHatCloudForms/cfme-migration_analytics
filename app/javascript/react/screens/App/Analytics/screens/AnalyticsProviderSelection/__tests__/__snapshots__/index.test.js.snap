// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Data collection screen should mount with mapStateToProps reduced 1`] = `
Object {
  "detailedDataSelected": false,
  "onCancelClick": [Function],
  "onContinueClick": [Function],
  "selectDetailedDataAction": [Function],
  "selectProvidersAction": [Function],
  "selectedProviders": Array [
    Object {
      "id": 1,
      "name": "Provider 1",
    },
  ],
  "summaryData": Object {
    "providers": Array [
      Object {
        "id": 1,
        "name": "Provider 1",
      },
      Object {
        "id": 2,
        "name": "Provider 2",
      },
    ],
  },
}
`;
